GoogleTestAdapter/Core/Runners/TestResultCollector.cs

```diff
@@ -95,14 +95,14 @@ private void CreateMissingResults(TestCase[] testCases, TestCase crashedTestCase
                 });
             }
             if (testCases.Length > 0)
-                _logger.DebugInfo($"{_threadName}Created {testCases.Length} test results for tests which were neither found in result XML file nor in console output");
+                _logger.DebugInfo($"{_threadName}{String.Format(Resources.CreatedTestResults, testCases.Length)}");
         }
 
         private void ReportSuspiciousTestCases(TestCase[] testCases)
         {
             string testCasesAsString = string.Join(Environment.NewLine, testCases.Select(tc => tc.DisplayName));
             _logger.DebugWarning(
-                $"{_threadName}{testCases.Length} test cases seem to not have been run - are you repeating a test run, but tests have changed in the meantime? Test cases:{Environment.NewLine}{testCasesAsString}");
+                $"{_threadName}{testCases.Length} {Resources.TestCaseNotRun}{Environment.NewLine}{testCasesAsString}");
```

Comment:
testCases.Length should be a fill-in in the localized string. NewLine maybe as well for consistency.